exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -53,6 +53,7 @@
   private int packetLength;
   protected int etherProtocol;
   protected int protocol;
+  protected boolean isCorrupt = false;
```

Comment:
Maybe add new field as last one, not in the middle?

Reply:
Fixed